brands/office/insurance.json

```diff
@@ -16,6 +16,7 @@
       "brand:wikidata": "Q87070013",
       "name": "A-Plan Insurance",
       "office": "insurance"
```

Comment:
Have you tried running build on your computer?  See https://github.com/osmlab/name-suggestion-index#installing (if it is unclear or insufficient - what is th problem with instructions?)

`       "office": "insurance"` is missing comma, as you added a new line (and in json last line and only last line is without ,)

Reply:
Oh sorry, I just didn't notice about that. Of course, I missed the comma, yes that's true. But it will be easy to fix...